sky/task.py

```diff
@@ -456,8 +456,25 @@ def from_yaml_config(
             task.set_outputs(outputs=outputs,
                              estimated_size_gigabytes=estimated_size_gigabytes)
 
+        # Experimental configs.
+        experimnetal_configs = config.pop('experimental', None)
+        cluster_config_override = None
+        if experimnetal_configs is not None:
+            cluster_config_override = experimnetal_configs.pop(
+                'config_overrides', None)
+            logger.debug('Overriding skypilot config with task-level config: '
+                         f'{cluster_config_override}')
+        assert not experimnetal_configs, ('Invalid task args: '
+                                          f'{experimnetal_configs.keys()}')
+
         # Parse resources field.
-        resources_config = config.pop('resources', None)
+        resources_config = config.pop('resources', {})
+        if cluster_config_override is not None:
+            assert resources_config.get('_cluster_config_overrides') is None, (
+                'Cannot set _cluster_config_overrides in both resources and '
```

Comment:
Should we raise an error if we find user specified `resources._cluster_config_overrides` as this is not allowed (according to the comment in the schema)?

Reply:
Good point! Lets keep the current one